### DISCUSSION TITLE
Pin an app ("Keep in Taskbar") by dragging it into Taskbar

### DISCUSSION THREAD

**Discussion by hencomb** [lawand-dot-io/taskbar#65] [Feature requests]
Hi there,

Great app!

Feature request to smooth out the experience:
* ability to drag an app onto the Taskbar - same as the Dock.

**Comment by lawand-dot-io** [comment]
@hencomb I'm glad you find Taskbar useful. The transparent background feature is already requested: https://github.com/lawand-dot-io/taskbar/discussions/42

**Comment by lawand-dot-io** [comment]
As for dragging an app into Taskbar, do you mean the ability to launch it? This is already possible by right clicking a window and selecting "Keep in Taskbar". Do you mean the same feature but instead of by right click it would be by dragging the app's icon into Taskbar?

**Comment by hencomb** [comment]
Hi there – thanks for the quick response!

Yes, the ability to drag an app into the Taskbar so that it stays there, the same that happens when you do that with the Dock.

**Comment by lawand-dot-io** [comment]
Ok thanks for the info, I understand what you mean now. As always, feature requests that get the most upvotes are prioritized. Will keep this one in mind though. Cheers
